addons/escoria-core/game/esc_logger.gd

```diff
@@ -0,0 +1,194 @@
+class ESCLoggerBase:
```

Comment:
One thing that we should get in the habit of doing is the ordering of code in a file, according to https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html#code-order

I've been trying to update our code whenever I can to follow this, but I'd strongly recommend a good starting point being any new files we add in, e.g. this one, the new autoload file, etc.